doc/manuals.jp/orion-api.md

```diff
@@ -2329,6 +2329,10 @@ the value of the "temperature" attribute (of type Number) is 23.4
 通知にカスタム・ペイロードが使用されている場合 (フィールド `payload` は対応するサブスクリプションにあります)、通知の
 `Ngsiv2-AttrsFormat` ヘッダに `custom` の値が使用されます。
 
+-   テキスト・ベースまたは JSON ペイロードが使用される場合 (つまり、 `payload` または `json` フィールドが使用される場合)、
+    `Ngsiv2-AttrsFormat` ヘッダは `custom` に設定されます。ただし、NGSI patch が使用される場合 (つまり、`ngsi` フィールド)、
+    通常の通知と同様に、`Ngsiv2-AttrsFormat: normalized` が使用されることに注意してください (通知形式が実際には同じである場合)
+
```

Comment:
I'm afraid I don't understand Japanese... but comparing the structure in https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#additional-considerations, maybe the paragraph in L2329-2331 should be removed?

As reference, this is the change that was done in this part:

![imagen](https://user-images.githubusercontent.com/1534240/219597843-9da336a3-cb2d-44fa-b5e7-70e99e3c035f.png)

Reply:
I made a mistake. I removed the old description.